specifications/active/aRMT-1.6.1.yml

```diff
@@ -0,0 +1,138 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.6.1
```

Comment:
is there a reason this was restored? was this ever registered on the server and MP?

Reply:
Yes this was registered on the server. The new topic was `cns_covid19_followup_v2`. I think this accidentally got deleted.